README.md

```diff
@@ -1,9 +1,52 @@
-## Google Authentification
+# AAD-Admin
 
-1. create an oauth 2.0 client id, used google cloud to generate one but im not sure how that will be done otherwise.
-   read this: https://developers.google.com/fit/android/get-api-key
+## Getting Started
 
-2. specify client id by including this line:
-<meta name="google-signin-client_id" content="YOUR_CLIENT_ID.apps.googleusercontent.com">
+1. **Create and setup your .env file**: Refer to .env.example
 
-3. added a new google-signin.tsx (not sure if this necessary, just following along with documentation)
+2. **Add Google OAuth 2.0 client to the Application**: in index.ts (src/pages/index.tsx) add the following to _AuthShowcase_
+
+   ```
+   <meta name="google-signin-client_id" content="YOUR_CLIENT_ID.apps.googleusercontent.com" />
```

Comment:
I think supplying the env var should be enough, we should not need to do this. For my own project with NextAuth I did not need to do this and it was able to auth through google.

Reply:
You are right, I removed and just had in .env and it worked.